packages/contract-tests/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "@odh-dashboard/tsconfig/tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true,
+    "allowImportingTsExtensions": true,
+    "module": "esnext",
+    "target": "es2021",
+    "moduleResolution": "bundler"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "**/*.test.ts", "**/*.spec.ts"]
+}
```

Why do you need any of these customizations?

```suggestion
{
  "extends": "@odh-dashboard/tsconfig/tsconfig.json"
}
```